src/survey.ts

```diff
@@ -146,18 +146,18 @@ export class SurveyModel extends SurveyElementCore
    * A survey instance that raised the event. Use `sender.data` to access survey results.
    * - `options.isCompleteOnTrigger`: `Boolean`\
    * Returns `true` if survey completion is caused by the ["complete" trigger](https://surveyjs.io/form-library/documentation/design-survey/conditional-logic#complete).
-   * - `options.showDataSaving(text?: string)`\
+   * - `options.showSaveInProgress(text?: string)`\
    * Call this method to indicate that the save operation is in progress. You can use the `text` parameter to display a custom message.
-   * - `options.showDataSavingError(text?: string)`\
+   * - `options.showSaveSuccess(text?: string)`\
    * Call this method to indicate that an error occurred during the save operation. You can use the `text` parameter to display a custom error message.
-   * - `options.showDataSavingSuccess(text?: string)`\
+   * - `options.showSaveSuccess(text?: string)`\
    * Call this method to indicate that survey results are successfully saved. You can use the `text` parameter to display a custom message.
-   * - `options.showDataSavingClear()`\
+   * - `options.clearSaveMessages()`\
    * Call this method to hide the save operation messages.
    *
    * For an example of how to use the methods described above, refer to the following help topic: [Store Survey Results in Your Own Database](https://surveyjs.io/form-library/documentation/handle-survey-results-store#store-survey-results-in-your-own-database).
    *
-   * > Do not disable the [`showCompletedPage`](https://surveyjs.io/form-library/documentation/surveymodel#showCompletedPage) property if you call one of the `options.showDataSaving...` methods. This is required because the UI that indicates data saving progress is integrated into the complete page. If you hide the complete page, the UI also becomes invisible.
+   * > Do not disable the [`showCompletedPage`](https://surveyjs.io/form-library/documentation/surveymodel#showCompletedPage) property if you call one of the `options.showSaveInProgress...` methods. This is required because the UI that indicates data saving progress is integrated into the complete page. If you hide the complete page, the UI also becomes invisible.
```

Comment:
```suggestion
   * > Do not disable the [`showCompletedPage`](https://surveyjs.io/form-library/documentation/surveymodel#showCompletedPage) property if you call one of the `options.showSave...` methods. This is required because the UI that indicates data saving progress is integrated into the complete page. If you hide the complete page, the UI also becomes invisible.
```